packages/fxa-payments-server/src/lib/apiClient.ts

```diff
@@ -275,6 +276,7 @@ export async function apiCapturePaypalPayment(params: {
 }> {
   const metricsOptions: Amplitude.EventProperties = {
     planId: params.priceId,
+    productId: params.productId,
```

Comment:
Can you update the `apiCapturePaypalPayment` test in `apiClient.test.ts` to assert that the Amplitude methods are called with the `productId`, similar to the `apiCreateSubscriptionWithPaymentMethod` method's test?

Reply:
Thanks for the review, I have updated the test accordingly.